### PR TITLE
feat: add dataSetId support to 3dModel [BND3D-1121] 

### PR DESCRIPTION
**Pull request by j-bjorne**

### CODE REVIEW

**Comment by andersfylling** [comment]
I can't find the documentation for this: https://docs.cognite.com/api/v1/#operation/create3DModels

Could you guide me in the right direction?

**Comment by j-bjorne** [comment]
https://github.com/cognitedata/service-contracts/pull/1388
I will not merge this before this one get's merged. I was just readying it :)

**Review by andersfylling — APPROVED** [review]
LGTM!

**Pull request merged**
